src/algebra/ring/basic.lean

```diff
@@ -841,6 +841,17 @@ end
 @[simp] theorem even_neg (a : α) : even (-a) ↔ even a :=
 dvd_neg _ _
 
+lemma odd.neg {p : α} (hp : odd p) : odd (-p) :=
```

Comment:
Since `even_neg` uses `a` instead of `p`, this probably should too.

Reply:
Not sure how I ended up with a `p` here. Fixed.